applications/asset_tracker_v2/src/modules/led_module.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**@file
+ *
+ * @brief   LED module.
+ *
+ * Module that handles LED behaviour.
+ */
+
+#ifndef LED_H__
+#define LED_H__
+
+#include <zephyr.h>
+#include <caf/led_effect.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define LED_PERIOD_NORMAL	500
```

Comment:
If `struct asset_tracker_led_effect` is unused move these three devices to led_state_defs and delete these file